mods/cnc/languages/rules/en.ftl

```diff
@@ -47,47 +47,46 @@ faction-gdi =
 faction-nod =
     .name = Nod
     .description = Brotherhood of Nod
-     The Brotherhood is a religious cult centered around their leader Kane
-     and the alien substance Tiberium. They utilize stealth technology
-     and guerilla tactics to defeat those who oppose them.
+     The Brotherhood is a religious cult centered around their leader, Kane,
+     and the alien substance Tiberium. They use stealth technology
+     and guerrilla tactics to defeat those who oppose them.
 
 ## defaults.yaml
 notification-unit-lost = Unit lost.
 notification-unit-promoted = Unit promoted.
 notification-building-captured = Building captured.
-notification-civ-building-captured = Civilian Building captured.
+notification-civ-building-captured = Civilian building captured.
 notification-structure-lost = Structure lost.
 
 ## aircraft.yaml
 actor-tran =
    .name = Chinook Transport
    .description = Fast Infantry Transport Helicopter.
-      Unarmed
+      Unarmed.
    .encyclopedia = The chinook is a flying transport capable of carrying a detatchement of infantry. In general, it is used to transport commandos or engineers for backline-destruction.
     Like all aircraft, it can be handy in a pinch to crush a commando by landing!
 
 actor-heli =
    .name = Apache Longbow
-   .description = Helicopter Gunship with Chainguns.
-      Strong vs Infantry, Light Vehicles and
-      Aircraft
-      Weak vs Tanks
+   .description = Helicopter Gunship with chainguns.
+      Strong vs. Infantry, Light Vehicles and Aircraft.
+      Weak vs. Tanks
```

Comment:
Missing full stop after "Weak vs. Tanks" - which seems to have been added elsewhere.